homeassistant/components/wsdot/strings.json

```diff
@@ -0,0 +1,51 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]",
+      "invalid_travel_time_id": "WSDOT TravelTimeId does not describe a valid Travel Time route {travel_time_id}",
+      "no_travel_time": "Travel Time route created without a Travel Time entity"
+    },
+    "step": {
+      "user": {
+        "title": "WSDOT setup",
+        "description": "Set up your Washington State Department of Transportation integration.",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "api_key": "Your WSDOT API Key."
+        }
+      }
+    }
+  },
+  "config_subentries": {
+    "travel_time": {
```

Comment:
by the way, just making sure, since you mentioned wsdot has a ton of services, is travel_time the best name? we can't change this once it has been released, so adding travel_time seems a little bit weird as it's more of a route/public transport stop